modules/dmrpp_module/data/get_dmrpp.in

```diff
@@ -115,50 +116,54 @@ BES.DefaultResponseMethod=POST
 # FONc handler specific parameters:
 #-----------------------------------------------------------------------#
 #Make sure this follows the netCDF-4 enhanced model
-FONc.ClassicModel=false
+#jhrg FONc.ClassicModel=false
 #Don't generate global attributes since we only
 #care about the variable value information.
-FONc.NoGlobalAttrs=true
+#jhrg FONc.NoGlobalAttrs=true
 
 #-----------------------------------------------------------------------#
 # HDF5 handler specific parameters:
 #-----------------------------------------------------------------------#
 # EnableCF: Groom the HDF5 data to follow the CF conventions
 #   (true,yes|false,no, defaults to false)
 #
+# HDF5 handler keys that are not the compiled-in defaults. jhrg 5/24/22
+H5.EnableCheckNameClashing=true
 H5.EnableCF=false
-H5.EnableCFDMR=true
 
-H5.DefaultHandleDimension=true
-H5.KeepVarLeadingUnderscore=false
-H5.EnableCheckNameClashing=true
-H5.EnableAddPathAttrs=true
-H5.EnableDropLongString=true
-H5.DisableStructMetaAttr=true
-H5.EnableFillValueCheck=true
-H5.CheckIgnoreObj=false
-H5.MetaDataMemCacheEntries=300
-H5.LargeDataMemCacheEntries=0
-H5.SmallDataMemCacheEntries=0
-#H5.CachePurgeLevel=0.2
-
-H5.EnableDiskMetaDataCache=false
-H5.EnableDiskDDSCache=false
-H5.DiskMetaDataCachePath=/tmp
-
-H5.EnableEOSGeoCacheFile=false
-H5.Cache.latlon.path=/tmp/latlon
-H5.Cache.latlon.prefix=l
-H5.Cache.latlon.size=20000
-
-H5.EnableDiskDataCache=false
-H5.DiskCacheDataPath=/tmp
-H5.DiskCacheFilePrefix=c
-H5.DiskCacheSize=10000
-H5.DiskCacheComp=true
-H5.DiskCacheFloatOnlyComp=true
-H5.DiskCacheCompThreshold=2.0
-H5.DiskCacheCompVarSize=10000
+# These are all the default values for the keys - based on the C++ code
+# and the default values see in HDF5RequestHandler.cc. jhrg 5/24/22
+#jhrg H5.EnableCFDMR=true
```

Comment:
Just because this is a default value is no reason not reiterate it. This is a new feature control, and poorly understood by anyone except @kyang2014 and possibly you.

Reply:
I disagree. More than you can know.

Reply:
ok